### PR TITLE
Libshell/Builtin: Improve spacing for the `time` builtin

### PR DESCRIPTION
**Pull request by ninadsachania**

### CODE REVIEW

File: Userland/Libraries/LibShell/Builtin.cpp [hunk 1]
```diff
@@ -1126,6 +1126,8 @@ ErrorOr<int> Shell::builtin_time(Main::Arguments arguments)
         iteration_times.add(static_cast<float>(timer.elapsed()));
     }
 
+    warnln(); // Tasteful spacing
```
Comment (nico) [inline Userland/Libraries/LibShell/Builtin.cpp:1129]: a) useless comment
b) no other `time` impl seems to print this line

**Review by nico — COMMENTED** [review]
First commit lg. Second commit feels a bit feng-shui-y (per https://github.com/SerenityOS/serenity/blob/master/CONTRIBUTING.md), but eh, without comment I suppose it's fine.

File: Userland/Libraries/LibShell/Builtin.cpp (hunk 1 above)
Comment (ninadsachania) [reply Userland/Libraries/LibShell/Builtin.cpp:1129]: a) OK, I'll remove that. FWIW, I copied the comment from the `fortune` utility.
b) The default GNU one sure does. Adding a blank line helps a user easily distinguish between the output of the command being profiled and the timing report.

Proof:
image

**Comment by nico** [comment]
TY!

**Pull request merged**